course_discovery/apps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -279,6 +303,35 @@ def test_contains_multiple_course_runs(self):
             }
         )
 
+    def test_contains_multiple_course_runs_edx_org_short_name_filter(self):
+        """
+        Verify contained course runs filtering on edX organization.
+        """
+        elastic_search_query_string = urllib.parse.urlencode({
+            'query': 'id:course*',
+            'course_run_ids': '{course_run_1_key},{course_run_2_key},{course_run_3_key}'.format(
+                course_run_1_key=self.course_run.key,
+                course_run_2_key=self.course_run_2.key,
+                course_run_3_key='course-v1:edly+Demo+Demo_Course'
+            )
+        })
+        elastic_search_query_string_with_org_filter = elastic_search_query_string + '&org=' + self.edx_org_short_name
```

Comment:
use key-value string formatting instead of concatenating strings with `+`

Reply:
Done